backend/lib/endpoints/posts.js

```diff
@@ -367,11 +367,17 @@ async function routes(app) {
       const { userId } = req;
       const { postId } = req.params;
       const [findErr, post] = await app.to(Post.findById(postId));
+      const [orgErr, org] = await app.to(User.findById(userId));
+
       if (findErr) {
         req.log.error(findErr, "Failed retrieving post");
         throw app.httpErrors.internalServerError();
       } else if (post === null) {
         throw app.httpErrors.notFound();
+      } else if (org) {
```

Comment:
Instead of line 377-383 something like (similar for update):

```suggestion
      } else if (userId.equals(author.id) || userId.equals(author.ownerId)) { 
        throw app.httpErrors.forbidden();
      }
```

I haven't tested the resolution but with current state, via Postman, I was able to update another org's post.

Reply:
when I changed it to this instead, and tried deleting, it gave me the 403 forbidden error

Reply:
I forgot the `!`. I'll make the B/E changes and push to this branch.

Reply:
I needed to also move the author query after checking errors related to fetching post.

But this seems to work. Confirmed can edit/delete when you should via F/E. Confirmed cannot edit/delete when you should not via Postman.